kie-bom/pom.xml

```diff
@@ -2745,7 +2745,7 @@
       </dependency>
       <dependency>
         <groupId>org.kie</groupId>
-        <artifactId>business-central-webapp</artifactId>
+        <artifactId>business-central-webapp-thorntail</artifactId>
```

Comment:
this seems to me redundant as the classifier is already thorntail

Reply:
@mareknovotny I totally agree but I didn't find a more appropriated name :thinking: 
`org.kie:business-central-webapp-thorntail:thorntail` wich is a bit confusing yes, let me thing a better name